src/main/java/frc/robot/subsystems/swerve/ModuleIO.java

```diff
@@ -33,7 +34,7 @@ public static class ModuleIOInputs {
     public double[] turnCurrentAmps = new double[] {};
 
     public double[] odometryDrivePositionsMeters = new double[] {};
-    public Rotation2d[] odometryTurnPositions = new Rotation2d[] {};
+    public NullableRotation2d[] odometryTurnPositions = new NullableRotation2d[] {};
```

Comment:
I also don't see it handling null odometryTurnPositions.

Reply:
that should be handled on L70